quipucords/api/migrations/0036_alter_credential_cred_type_alter_source_source_type.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.1 on 2023-09-14 23:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("api", "0035_alter_detailsreport_sources"),
```

Comment:
You need to rebase this PR against upstream/main and update this migration dependency (also rename it to match current migration numbering